flow-report/assets/styles.css

```diff
@@ -0,0 +1,145 @@
+.App {
+  display: grid;
+  grid-template-areas: "side report";
+  grid-template-columns: min-content auto;
+  min-height: 100vh;
+  max-width: 100vw;
+}
+.App a {
+  color: var(--color-gray-800);
+  text-decoration: none;
+}
+
+.Hbar {
+  height: 1px;
+  width: 100%;
+  background-color: var(--color-gray-300);
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.Sidebar {
+  border-right: 1px solid var(--color-gray-300);
+}
+
+.Sidebar_current {
+  background-color: var(--color-gray-200);
+  font-weight: bold;
+}
+
+.SidebarSectionTitle {
+  height: 16px;
+  font-size: 11px;
+  margin: 32px 16px 0px 16px;
+  color: var(--color-gray-700);
+}
+
+.SidebarHeader {
+  margin: 16px;
+}
+.SidebarHeader_title {
```

Comment:
BEM is typically `__` to add more space between labels, any particular affinity for single?